Civi/Api4/Service/Spec/Provider/PriceFieldValueCreationSpecProvider.php

```diff
@@ -29,6 +29,8 @@ class PriceFieldValueCreationSpecProvider implements Generic\SpecProviderInterfa
   public function modifySpec(RequestSpec $spec) {
     // Name will be auto-generated from label if not supplied
     $spec->getFieldByName('name')->setRequired(FALSE);
+    // Ensure that label is required this matches v3 API but doesn't match DAO because form fields allow for NULLs
+    $spec->getFieldByName('label')->setRequired(TRUE);
```

Comment:
@seamuslee001 so I'm inclined to go with it not being required in either api - I think the reason it was required was nasty munging behaviour. If that's hard to deal with I'm OK with apiv4 not matching v3 - as long as v4 is 'right'

(am also OK punting that for master)

Reply:
I have no idea but it seems it has been required in the API since v4.5 https://github.com/civicrm/civicrm-core/commit/e825f029159d81d13f0f0f0a47139fc09706983a so it would feel very strange to break that but maybe its ok for v4. Also the JIRA mentioned in that commit looks wrong https://issues.civicrm.org/jira/browse/CRM-14788. I would say keep label required for the moment and can manage in master